pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1325,6 +1325,12 @@ private void checkOwnershipAndCreatePersistentTopic(final String topic, boolean
                 });
     }
 
+    @VisibleForTesting
+    public void createPersistentTopic0(final String topic, boolean createIfMissing,
```

Comment:
why don't use `BrokerService#getTopic`?

Reply:
yes we can use the method, but when I wrote the test, I chose this method